lib/motion.ts

```
@@ -0,0 +1,204 @@
+  useEffect(() => {
+    if (!ref.current) return;
+
+    return inView(ref.current, () => {
+      setIsInView(true);
+      return () => {
+        if (!options?.once) {
+          setIsInView(false);
+        }
+      };
+    }, { 
+      margin: options?.margin || '0px'
```

ESLint

[error] Strings must use doublequote.

(quotes)